### DISCUSSION TITLE
Negative minimum lengths to add padding to the right of tokens

### DISCUSSION THREAD

**Discussion by joshpetit** [polybar/polybar#2788] [Ideas]
Right now if we do
```
label = %date:15%
```
Then it will do padding to the left of of the label

![image](https://user-images.githubusercontent.com/59021155/185800065-6dc28c87-8cdf-4e81-a906-2be6ab9ba7e2.png)

Adding negatives to enable us to do padding to the 
For example:
```
label =  %date:-15%
```
Would make the `label` a minimum of 15 characters, and if it is less than that then it'll add spaces as padding to the right of the label.

**Comment by patrick96** [comment]
Thanks! I created a feature request issue: #2789
